### PR TITLE
add cloudsecurity_cdr category

### PR DESCRIPTION
**Pull request by CohenIdo**
- https://github.com/elastic/security-team/issues/8667

### CODE REVIEW

File: categories/categories.yml [hunk 1]
```diff
@@ -113,6 +113,8 @@ categories:
         title: "DNS"
       edr_xdr:
         title: "EDR/XDR"
+      cloudsecurity_cdr:
+        title: "CDR"
```
Comment (jsoriano) [inline categories/categories.yml:117]: @nimarezainia are we ok with adding this category as a subcategory of `security` category?

**Review by jsoriano — COMMENTED** [review]
LGTM, waiting for confirmation from product.

**Comment by CohenIdo** [comment]
> LGTM, waiting for confirmation from product.
cc: @snehsach19

**Review by nimarezainia — APPROVED** [review]

**Review by nimarezainia — APPROVED** [review]

File: categories/categories.yml (hunk 1 above)
Comment (snehsach19) [reply categories/categories.yml]: lgtm

**Pull request merged**